.github/templates/linux_ci_workflow.yml.j2

```
@@ -348,6 +348,8 @@ jobs:
         run: |
           env | grep '^GITHUB' > "/tmp/github_env_${GITHUB_RUN_ID}"
       - name: Test PyTorch
+        env:
+          BUILD_ENVIRONMENT: !{{ build_environment }}-${{ matrix.config }}
```

the bazel.yml.j2 has similar blocks define, do you think we also need to set the same thing there?

---

not necessarily for now because bazel environments are all checked against *-bazel-*. so I think we are good. but good point I think we should do so for windows/macos/special_cases in the future

---

```suggestion
          BUILD_ENVIRONMENT: !{{ build_environment }}-$TEST_CONFIG
```

why not use $TEST_CONFIG?